templates/table.js.tera

```diff
@@ -303,6 +312,15 @@ function renderMarkdownDescription() {
     innerDescription.innerHTML = converter.makeHtml(innerDescription.dataset.markdown);
 }
 
+function precision_formatter(precision, value) {
+    value = parseFloat(value)
+    if (1 / (10 ** precision) < value) {
+        return value.toString()
+    } else {
+        return value.toExponential(precision)
+    }
+}
+
```

Comment:
Why the repetition with the block above? I guess it will be possible to have it just once?

Reply:
Argh, simply forgot to delete the first section. Should be fixed now!